### ISSUE TITLE
[BUG] Issue with "ddev snapshot restore" - hyphens in snapshot name seem to be an issue

### ISSUE THREAD

**Issue by ultimike** [biati-digital/vscode-ddev-manager#10]
I created a snapshot with the name `after-class-12` using DDEV Manager, but when I went to restore it (again, using DDEV Manager), it failed.

It might be related to the name I gave the snapshot, because when I went to restore it from the UI, the hyphens were removed from the name:

<img width="670" alt="Screenshot 2023-09-22 at 2 53 05 PM" src="https://github.com/biati-digital/vscode-ddev-manager/assets/1727889/dcafcaa4-66a3-4b0c-bd4a-00e0e0d41400">

The error I received when trying to restore is: `Error restoring snapshot, check the extension output log for more information.` The logs show:

```
[error] Failed to restore snapshot afterclass12_20230922145223 for project pmdsum2023: no snapshot found for name afterclass12_20230922145223: snapshot afterclass12_20230922145223 not found in /Users/michael/sites/pmdsum2023/.ddev/db_snapshots
```

I was able to successfully restore the snapshot from the command line using: `ddev snapshot restore after-class-12_20230922145223`

**System (please complete the following information):**

- OS: Mac OS X Ventura 13.5.2
- DDEV version: 1.22.1
- Docker provider: Colima
- VSCode version: 182.2
- DDEV Manager version: 1.3.1

**Comment by biati-digital** [comment]
Thank's for letting me know, I've confirmed the bug. Will be fixed in the next update.

**Comment by biati-digital** [comment]
btw in the meantime you can use as name `after class 12` instead of `after-class-12` and will work correctly.

**Comment by biati-digital** [comment]
This is fixed in the latest update 1.3.2
